ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js

```diff
@@ -104,6 +104,32 @@ module.exports = Marionette.LayoutView.extend({
         user.getQuerySettings().set('type', 'text')
         break
       case 'custom':
+        let sorts =
```

Comment:
Lines 107-115 would probably be better as a ternary

Reply:
I would rather not turn this into a ternary because it's pretty repetitive:
`let sorts =`
` this.model.get('querySettings') &&`
`this.model.get('querySettings').sorts`
`? this.model.get('querySettings').sorts.map(sort => ({`
`attribute: sort.split(',')[0],`
`direction: sort.split(',')[1],`
`}))`
`: this.model.get('querySettings') &&`
`this.model.get('querySettings').sorts`
Unless you have a better way of doing it that I overlooked @samuelechu?